packages/ui/components.json

```diff
@@ -3,7 +3,7 @@
   "style": "default",
   "rsc": true,
   "tailwind": {
-    "config": "tailwind.config.js",
+    "config": "@penumbra-zone/tailwind-config",
```

Comment:
I actually don't know how this setting works or what it does, so I'm not sure how to test that this change worked...